packages/elemental/elemental/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,46 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -1,28 +1,37 @@
+ image:
+   empty: rancher/pause:3.1
+-  # elemental-operator image is also built on:
+-  #   registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-operator
+-  repository: "quay.io/costoolkit/elemental-operator"
+-  tag: v1.2.5
++  repository: "registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-operator/5.3"
++  tag: "1.2.5"
+   imagePullPolicy: IfNotPresent
++
+ seedImage:
+-  repository: "quay.io/costoolkit/elemental-seedimage"
+-  tag: v1.2.5
++  repository: "registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/seedimage-builder/5.3"
++  tag: "1.2.5"
+   imagePullPolicy: IfNotPresent
++
++channel:
++  repository: "registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-teal-channel/5.3" 
++  tag: latest
```

Comment:
Also, generally `latest` is not recommended. Can you please put the specific tag ?

Reply:
The channel here is just a container image carrying a list of URL with updated Elemental OSes, so we wanted it to keep updating... but your comment raised some discussion inside the team, and you are likely right: we need a specific tag here, but then we need to rethink how to provide OS updates 🤔 
We'll need some time to address this

Reply:
You could update the image to the new tag and release a new Chart in `rancher/charts` by upgrading the version in `package.yaml`

Reply:
We will check the same image tag regularly and we may required to update it... we anyway need a fixed tag to match the specific Chart version (and so Elemental release). Updated to fixed tag 👍🏼